api/base/serializers.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, related_view=None, related_view_kwargs=None, self_view=None,
         self.related_meta = related_meta
         self.self_meta = self_meta
         self.always_embed = always_embed
+        self.source_field = source_field
```

Comment:
Also, it would be useful to document what that does in the docstring, as it isn't obvious how it's used..